go.mod

```diff
@@ -18,18 +18,22 @@ require (
 	github.com/opencontainers/image-spec v1.0.2-0.20190823105129-775207bd45b6
 	github.com/pkg/errors v0.9.1
 	github.com/psampaz/go-mod-outdated v0.7.0
-	github.com/saschagrunert/go-modiff v1.2.0
-	github.com/sendgrid/rest v2.6.1+incompatible
-	github.com/sendgrid/sendgrid-go v3.6.4+incompatible
+	github.com/saschagrunert/go-modiff v1.2.1
+	github.com/sendgrid/rest v2.6.2+incompatible
+	github.com/sendgrid/sendgrid-go v3.7.0+incompatible
 	github.com/sirupsen/logrus v1.7.0
-	github.com/spf13/cobra v1.0.0
+	github.com/spf13/cobra v1.1.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
 	github.com/yuin/goldmark v1.2.1
-	golang.org/x/net v0.0.0-20200930145003-4acb6c075d10
+	golang.org/x/crypto v0.0.0-20201012173705-84dcc777aaee // indirect
+	golang.org/x/net v0.0.0-20201016165138-7b1cca2348c0
 	golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43
-	google.golang.org/api v0.32.0
+	golang.org/x/sys v0.0.0-20201016160150-f659759dc4ca // indirect
+	golang.org/x/tools v0.0.0-20201016202518-de28cda03d9c // indirect
```

Comment:
Let's remove those indirect dependencies and run `go mod tidy`

Reply:
I've removed it. TBH I don't know what pulled those, but they are now gone.